prime-router/metadata/hl7_mapping/receivers/STLTs/CA/CA.yml

```diff
@@ -115,16 +115,28 @@ elements:
   # SFT-1-1
```

Comment:
Like RADx MARS, these changes intend to preserve the behavior for CA, despite formerly overridden elements being adjusted or deleted.

Reply:
This file can actually be deleted. I left it in case we needed to rollback, but it's no longer needed.

Reply:
Glad to hear it, we were wondering, but weren't sure. I think it's still referenced in some translation tests, I'll update those and remove this in a new commit.

Reply:
It is also referenced in `0150-CA-Add-FullELR-SaphireStage-Receiver.yml`. Is that an issue? I can update those files to point to `CA-receiver-transform`, but will we need to update any settings on staging or prod?

Reply:
Small difference, `software-vendor-name` was not present in `CA-receiver-transform`, so SFT.1 changes in one of the translation tests by switching the transforms. I assume that's fine, just pointing it out in case.